model.md

```diff
@@ -0,0 +1,86 @@
+|        |                |                          |                    | studentId not found                               | throw error ('student not found')                                  |
+|        |                |                          |                    | cohortId not found                                | throw error ('cohort not found')                                   |
+|        |                |                          |                    | cohortStudentSize >= cohort.capacity              | throw error ('cohort capacity exceeded')                           |
+|        |                |                          |                    |                                                   |                                                                    |
+|        |                | removeStudentFromCohort  | studentId:number,  | inputIds match existing Ids                       | foundStudent.cohortId = null, return foundStudent                  |
+|        |                |                          | cohortId:number    |                                                   |                                                                    |
+|        |                |                          |                    | studentId not found                               | throw error ('studentId not found')                                |
+|        |                |                          |                    | cohortId not found                                | throw error ('cohortId not found')                                 |
+|        |                |                          |                    |                                                   |                                                                    |
+|        |                |                          |                    | foundStudent.cohortId !== inputCohortId           | throw error ('student not in passed cohort')                       |
+|        |                |                          |                    |                                                   |                                                                    |
+|        |                |                          |                    |                                                   |                                                                    |
+|        | Student        |                          |                    |                                                   |                                                                    |
+|        |                | details                  |                    |                                                   | return { ... student properties }                                  |
+|        |                |                          |                    |                                                   |                                                                    |
```

Comment:
This table has to be one of the cleanest and well put together .md's Ive seen in awhile. Nice work.

Reply:
Thank you! I have Prettier to thank for that.

Reply:
@Uprising5034 oh my god galaxy brain time